### ISSUE TITLE
Can't replicate the training procedure

### ISSUE THREAD

**Issue by zTsugumi** [swagshaw/XLSR-Mamba#5]
Hi @swagshaw,  

I'm trying to replicate the training of XLSR-Mamba for ASV 2021 LA assessment. However the result models always have high EER (2%) on the given eval dataset. I have used a slightly different environments to suit my project as follows:
* python == 3.10.18
* torch == 2.8.0
* librosa == 0.11.0
* mamba_ssm == 2.2.6.post3

I have tried training the model with DDP on 8 x A100 GPUS with the learning rate of 1e-6, 2e-6; batch size of 4, 8. All other settings follows your paper. Note that, I have successfully loaded your pretrained model so I think my code of the model architecture is not a problem. I doubt that it is the DDP that causes the degration. It would be great if you can give me some insights and advices to my problem. 

Kind regards,
Gumi

**Comment by swagshaw** [comment]
Hi Gumi,

Thanks for your interest in our work. However, I did not use DDP, so I can not explain it to you. Maybe you can try training it in single GPU. For me, I use A5000 24G.

Best,
Austin

**Comment by zTsugumi** [comment]
Thanks for the quick response. However, I tried with single GPU but the same behaviour persists without any improvement.
